docs/luigi-core-api.md

```diff
@@ -644,3 +654,80 @@ Hides the app loading indicator.
 **Meta**
 
 -   **since**: 0.6.4
+
+#### showAlert
+
+Shows an alert.
+
+##### Parameters
+
+-   `settings` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** the settings for the alert
```

Comment:
Some of the sentences do have dots at the end some don't - but I think it may be this rendering problem you told me about. Also, it's up to you to have / not have them as long as they are consistent :)

Reply:
Yes, I tried to add the dots but I get an error from the script which renders the docu 🙁

Reply:
So, I removed the dots for consistency.